doc/sphinx/language/core/sections.rst

```diff
@@ -3,57 +3,30 @@
 Section mechanism
 -----------------
 
-Sections create local contexts which can be shared across multiple definitions.
+Sections create naming scopes that permit creating section-local definitions that can
```

Comment:
I wouldn't call them "naming scopes" because they do not affect the naming of the things that are declared within. Why remove the use of "local context" in this case? It seemed quite appropriate.

Reply:
Each goal at each step in a proof has its own local context.  The section contributes items only to the initial local context of a proof.  "naming scope" a [well-known term](https://en.wikipedia.org/wiki/Scope_(computer_science)) that I think fits very well, though I should probably say "Sections are naming scopes".  Sections are not local contexts, which might be inferred from "Sections create local contexts".

Does that make sense?

Reply:
>  "naming scope" a [well-known term](https://en.wikipedia.org/wiki/Scope_(computer_science))

This is precisely the reason why I'm worried about the use of this name: sections do not create naming scopes. That's what I tried to convey in my previous comment. What about "Sections create a shared initial local context that will be extended in definitions and proofs."?

Reply:
Sorry, I was actually confusing "naming scope" with "namespace". Still the "naming" part is a bit misleading. What about keeping "scope" but removing "naming" then? E.g., "Sections create a scope in which all local contexts will include a shared list of local declarations".